### ISSUE TITLE
Image size

### ISSUE THREAD

**Issue by leonardorame** [jpambrun/dcmj2k-streaming-viewer#1]
Hi Jean-Francois, I wonder how did you manage to generate those tiny (~20kb) files in your demo. 

I compressed CT images using gdcmconv from ~500kb to ~160kb, did you use lossy compression?

**Comment by jpambrun** [comment]
Yes and no. I used Lossless streams with 3 embedded lossy quality layers. Initially, only the first low quality layer is downloaded for the entire series to enable fast navigation, but when the user stops for mote than 100 ms on a particular slice, additional layers are progressively downloaded until losslessness is achieved. This update process usually takes about a second even if the "pre-fetching" process isn't completed.

As you noted, the lossless stream is usually about 160kb.

**Comment by leonardorame** [comment]
Very interesting, hopefully this can be added to the WADOImageLoader.
